tracer/test/snapshots/NetActivitySdkTests.verified.txt

```diff
@@ -26,38 +26,46 @@
     }
   },
   {
-    TraceId: Id_1,
-    SpanId: Id_3,
+    TraceId: Id_3,
+    SpanId: Id_4,
     Name: Samples.NetActivitySdk.server,
     Resource: Ctor2,
     Service: Samples.NetActivitySdk,
     Type: web,
-    ParentId: Id_2,
```

Comment:
Ctor2 should have a parent ID that matches the ActivityContext you passed in. I think you'll need to create a new `SpanContext` object that represents the parent when creating this span

Reply:
I've updated it, but looks like it's putting in the wrong values for the ParentID from the JSON of the spans so I'll have to dig into this a bit further

Reply:
Is this outdated ? If not, I don't understand. Cause here we now have a `Id_5`, that is not present in the chunk, so I assume it's the context that was passed along, no?

Reply:
Yeah that `Id_2` is outdated now, I updated it yesterday and just forgot to update this

It's the `Id_5` which is mapping to the context that was passed to it now